dashboard/app/models/programming_expression.rb

```diff
@@ -186,8 +187,10 @@ def summarize_for_edit
       id: id,
       key: key,
       name: name,
+      blockName: block_name,
       category: category,
       programmingEnvironmentName: programming_environment.name,
+      blocklyProgrammingEnvironment: programming_environment.editor_type == 'blockly',
```

Comment:
this would be more flexible if we passed `programming_environment.editor_type`, so we could later support other editor types. do we want that flexibility, or will we only care about whether or not we're using blockly?

Reply:
I waffled on this and don't have a strong opinion. I can pass down the editor type instead